core/tauri/Cargo.toml

```diff
@@ -115,6 +115,7 @@ objc = "0.2"
 [target."cfg(windows)".dependencies]
 dunce = "1"
 webview2-com = "0.19.1"
+windows-sys = { version = "0.52.0", features = [ "Win32_Foundation", "Win32_UI_WindowsAndMessaging" ] }
```

Comment:
no need to add `windows-sys` dependencies, we already depend on its bigger brother `windows` crate